### DISCUSSION TITLE
The problem that the latest version cannot be called in real time

### DISCUSSION THREAD

**Discussion by tlerbao** [deployphp/deployer#3105] [Help needed]
After I successfully deploy the project using dep deploy -vvv, the access project is still the previous version, what is the reason?

For example: the latest version is 10, but the version 9 or 8 is called, there is no problem when keep_releases = 1 is set.

**Comment by antonmedv** [comment]
I didn’t get it. Please explain.

**Reply by tlerbao** [reply]
https://deployer.org/docs/7.x/avoid-php-fpm-reloading Solved, thanks.
